zebra/zebra_rib.c

```diff
@@ -3248,6 +3249,9 @@ void rib_delete(afi_t afi, safi_t safi, vrf_id_t vrf_id, int type,
 		rib_delnode(rn, same);
 	}
 
+	if (connected_down)
```

Comment:
Could we have a comment about this, since it's sort of a subtle change?

Reply:
done